packages/xdl/src/createWebpackCompiler.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk';
+import formatWebpackMessages from 'react-dev-utils/formatWebpackMessages';
+import clearConsole from 'react-dev-utils/clearConsole';
+import * as ProjectUtils from './project/ProjectUtils';
+const CONSOLE_TAG = 'webpack';
+
+function log(projectRoot, message) {
+  ProjectUtils.logInfo(projectRoot, CONSOLE_TAG, message);
+}
+
+function logWarning(projectRoot, message) {
+  ProjectUtils.logWarning(projectRoot, CONSOLE_TAG, message);
+}
+
+function logError(projectRoot, message) {
+  ProjectUtils.logError(projectRoot, CONSOLE_TAG, message);
+}
+
+function printInstructions(projectRoot, appName, urls, useYarn) {
+  const _log = message => log(projectRoot, message);
+
+  _log(` `);
+  _log(`You can now view ${chalk.bold(appName)} in the browser.`);
+  _log(` `);
+  if (urls.lanUrlForTerminal) {
+    _log(`  ${chalk.bold('Local:')}            ${urls.localUrlForTerminal}`);
+    _log(`  ${chalk.bold('On Your Network:')}  ${urls.lanUrlForTerminal}`);
+    _log(` `);
```

Comment:
These logs look really sparse in Dev Tools because each line is a separate message:
<img width="645" alt="Screenshot 2019-04-11 at 11 37 54" src="https://user-images.githubusercontent.com/497214/55943506-63ea9300-5c4f-11e9-9f26-15d289bdbc00.png">

Perhaps you can do something like call `_log( <a long message for the whole thing with newlines in it> )` instead to make it look nicer. Especially logging a `' '` (single space) looks bad in Dev Tools, you can replace that by adding newlines to the message.

Reply:
Also there are no webpack build progress bars/messages visible in Dev Tools right now.